code-challenges.test.js

```diff
@@ -45,6 +81,33 @@ const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 
 // b) Create the function that makes the test pass.
 
+describe ("returnCap", () => {
+    it("should return the array capitilized", () => {
+    expect(returnCap(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+     expect(returnCap(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+})
+})
+
+const returnCap = (value1) => {
+    
+    return value1.map((value) => {
+        return value.charAt(0).toUpperCase() + value.slice(1).toLowerCase()
+    })
+}
+
+
+
+//psuedocode:
+// create a test for my arrays to pass, use decribe " returnCap" , it "should return the array capitilized", expect.
+// create the function to make it work
+// i will have to use map to make a new array, then return that array with charAt(0) to access the first place in array and only capitolize that.
+// use slice(1) to keep the rest lowercase not sure if needed---
```

Comment:
Yes! You did awesome at tackling this problem and yes you will need to use an additional method that will leave the rest of the string lowercase. There are many ways but this one worked and looks really good.